packages/contracts-bedrock/contracts/legacy/LegacyMessagePasser.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Semver } from "../universal/Semver.sol";
+
+/**
+ * @custom:legacy
+ * @custom:proxied
+ * @custom:predeploy 0x4200000000000000000000000000000000000000
+ * @title LegacyMessagePasser
+ * @notice The LegacyMessagePasser was the low-level mechanism used to send messages from L2 to L1
+ *         before the Bedrock upgrade. It is now deprecated in favor of the new MessagePasser.
+ */
+contract LegacyMessagePasser is Semver {
+    /**
+     * @notice Mapping of sent message hashes to boolean status.
+     */
+    mapping(bytes32 => bool) public sentMessages;
+
+    /**
+     * @custom:semver 0.0.1
+     */
+    constructor() Semver(0, 0, 1) {}
+
+    /**
+     * @notice Passes a message to L1.
+     *
+     * @param _message Message to pass to L1.
+     */
+    function passMessageToL1(bytes memory _message) external {
+        sentMessages[keccak256(abi.encodePacked(_message, msg.sender))] = true;
```

Comment:
We should add back the event that never was here
https://github.com/ethereum-optimism/optimism/blob/009939e0143e15c42919521bd9004cb69a05ebbe/packages/contracts/contracts/L2/predeploys/iOVM_L2ToL1MessagePasser.sol#L12
lol, since it was left out previously I'm ok with it being a different event

Reply:
I would prefer not to add the event. We're never going to use this contract again and we never emitted it previously. I don't want to change the behavior of a legacy contract unless strictly necessary.

Reply:
Actually, this is a good point. Maybe legacy contracts don't need semver.

Reply:
`LegacyERC20ETH` isn't getting Semver which is inconsistent

Reply:
Ok started discussion in slack, can figure it out there